src/Main.java

```diff
@@ -1,5 +1,71 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        Main main = new Main();
+        System.out.println(main.missingChar("kitten", 1));
+        System.out.println(main.missingChar("kitten", 0));
+        System.out.println(main.missingChar("kitten", 4));
+        System.out.println("------------------------------------");
+        System.out.println(main.sumDouble(1, 2));
+        System.out.println(main.sumDouble(3, 2));
+        System.out.println(main.sumDouble(2, 2));
+        System.out.println("------------------------------------");
+        System.out.println(main.intMax(1, 2, 3));
+        System.out.println(main.intMax(1, 3, 1));
+        System.out.println(main.intMax(3, 2, 1));
+        System.out.println("------------------------------------");
+        System.out.println(main.nearHundred(93,100));
+        System.out.println(main.nearHundred(90,100));
+        System.out.println(main.nearHundred(89,100));
+        System.out.println("------------------------------------");
+        System.out.println(main.notString("candy"));
+        System.out.println(main.notString("x"));
+        System.out.println(main.notString("not bad"));
+    }
+
+    /**
+     * Не  знаю точно как надо было , но нашла готовую функцию использая обьект класса StringBuilder ,
+     * возможно, надо было черезе substring ,
+     * тогда переделаю
+     **/
+
+    public StringBuilder missingChar(String str, int n) {
+        StringBuilder newStr = new StringBuilder(str);
```

Comment:
No, this is valid approach. 
You could have also used substrings. 
str.substring(0, n) to get the first half
str.substring(n+1, str.lenght() - 1) for the second half and then join those two strings. 
No need to redo this, it's totally fine